### PR TITLE
GH Templates

### LINKED ISSUE #13: decide on templates to use
following from http://github.com/librariesio/awesome-github-templates we should decide on what templates we should use, how they;re worded etc. to guide contributors into good practice WRT bug, feature and enhancement req. and PRs.

### PR DESCRIPTION
**Pull request by BenJam**
First draft of templates for contributors. 

* add issue and PR templates
* change links in CONTRIBUTNG.md to reference the these templates
* update links to issue trackers across the org

Closes #13

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Thanks for starting a pull request. This template should help guide your through the process of creating a pull request for review:
```
Comment (andrew) [inline pull_request_template.md]: This needs a similar line to the issue template:

> This template should guide you through the process of submitting a bug, enhancement or feature request.

Comment (BenJam) [inline pull_request_template.md]: how's this:

>Thanks taking the time to contribute to Libraries.io. This template should help guide your through the process of creating a pull request for review:

Comment (andrew) [inline pull_request_template.md]: Sorry I copy-pasta failed there, I meant to mention the bit about erasing parts:

> Please erase any part of this template that is not relevant to your issue.

**Comment by andrew** [comment]
Noice, these can be added to all the repos, perhaps in the `.github` folder?

**Comment by BenJam** [comment]
yup. will do once I know all the links in docs.libraries.io are working. Just looking now.

**Pull request merged**